src/languages/swift.js

```diff
@@ -399,11 +391,12 @@ export default function(hljs) {
   // https://docs.swift.org/swift-book/ReferenceManual/Declarations.html#ID375
   // https://docs.swift.org/swift-book/ReferenceManual/Declarations.html#ID379
   const INIT_SUBSCRIPT = {
-    className: 'function',
-    match: /\b(subscript|init[?!]?)\s*(?=[<(])/,
-    keywords: {
-      keyword: "subscript init init? init!",
-      $pattern: /\w+[?!]?/
+    match: [
+      /\b(?:subscript|init[?!]?)/,
+      /\s*(?=[<(])/,
+    ],
+    className: {
+      1: "keyword"
```

Comment:
I wonder if this shouldn't be `title.function` instead of keyword?

Reply:
If you mean `subscript` and `init`, those are just keywords. Subscripts and initializers are type members that don't have a title.